.github/workflows/codeql-analysis.yml

```diff
@@ -7,10 +7,10 @@ name: "CodeQL"
 
 on:
   push:
-    branches: [master]
+    branches: *
   pull_request:
     # The branches below must be a subset of the branches above
-    branches: [master]
+    branches: *
```

Comment:
keep only master; there are different configs needed for other versions (3.0 and 2.7 require Java SE 8 for build; CodeQL would be nice to have there; 2.6 (and older) should be completely excluded)

Reply:
note that 2.x do use ant for the build (see ie travis config in the branch or [specific build instructions](https://wiki.eclipse.org/EclipseLink/Building#Build))

for 2.6 and older there is only very limited support; it does not make sense to consume too much resources for that these days

Reply:
> note that 2.x do use ant for the build (see ie travis config in the branch or [specific build instructions](https://wiki.eclipse.org/EclipseLink/Building#Build))

Hope CodeQL can work with Ant.

> for 2.6 and older there is only very limited support; it does not make sense to consume too much resources for that these days

Agree. Would it make sense to fix severe security issues if CodeQL finds them?

Reply:
those would be present in master/3.0 and 2.7  as well. Everything going to 2.6 must go through newer versions first.